### PR TITLE
Fix for workflow_lint to work with params/collections and add it to docs

### PR DESCRIPTION
**Pull request by simonbray**
@bernt-matthias here is the fix for `workflow_lint`.

I think I'll keep this PR open for now and continue adding any other necessary fixes, rather than open lots of small PRs.

### CODE REVIEW

**Comment by jmchilton** [comment]
Thanks so much. We should just merge this - it looks solid! Lots of little PRs are good IMO.

File: docs/_running_intro.rst [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+    the latest version from PyPI or bioconda. To install the development versions
+    of Planemo and galaxy-tool-util required, run the following commands:
+
+    ::
+
+        $ # install dev version of planemo
+        $ pip install https://github.com/galaxyproject/planemo/archive/master.tar.gz
+        $ # install dev version of galaxy-tool-util
+        $ git clone https://github.com/galaxyproject/galaxy.git
+        $ pip install -e galaxy/packages/tool_util
```
Comment (jmchilton) [inline docs/_running_intro.rst:14]: I published the latest release to PyPI if that helps at all. https://pypi.org/project/galaxy-tool-util/21.1.0.dev4/

Comment (simonbray) [reply docs/_running_intro.rst]: Yes it does, thanks a lot! You can go ahead and merge this if you like.

**Comment by jmchilton** [comment]
Thanks so much - keep up the awesomeness!

**Pull request merged**